### DISCUSSION TITLE
Integration with MongoDB Charts

### DISCUSSION THREAD

**Discussion by demSly** [zitadel/zitadel#3005] [Q&A]
Hi,
I need to use Zitadel to manage access to some MongoDB Charts.
Can you help me configure it? This is how it looks like:

<img width="700" alt="Screenshot 2022-01-14 at 14 27 38" src="https://user-images.githubusercontent.com/43607658/149522857-ef5d0395-0171-484c-9019-7d9f959578c7.png">

**Comment by eliobischof** [comment]
Hi @demSly
Thanks for the question. I'll try it asap and come back to you soon.

**Reply by fforootd** [reply]
In my mind it should be feasible when a client request a JWT from ZITADEL as access_token.

In that case the JWKS URL https://api.zitadel.ch/oauth/v2/keys in combination with the AUD claim should work.

**Reply by eliobischof** [reply]
I could make it work using the way @fforootd suggested.

Configure https://api.zitadel.ch/oauth/v2/keys as JWKS URL and set your ZITADEL projects app client id as the audience.
In your ZITADEL app, configure the auth token type to be JWT.
![image](https://user-images.githubusercontent.com/12727842/149972158-18f36d55-4f9a-41f8-9020-66e3ff632a42.png)

@demSly Please let us know if it works for you this way.

Btw: A detailed integration guide is coming soon #3041

**Reply by mffap** [reply]
@demSly does that work for you?

**Reply by eliobischof** [reply]
@demSly the integration guide mentioned above is now online https://docs.zitadel.ch/docs/guides/integrations/authenticated-mongodb-charts
